api/src/test/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/PReLUTest.kt

```diff
@@ -77,12 +79,11 @@ class PReLUTest : LayerTest() {
             alphaTensor.close()
         }
 
-        val expected = getFloatArrayOfShape(input.shape).cast3D<FloatArray>()
```

Comment:
it was used to create 3d array which then has been cast - replaced by explicit creation of array.